### ISSUE TITLE
App store links in README broken; is the app still available?

### ISSUE THREAD

**Issue by bobwhitelock** [todotxt/todo.txt-android#507]
Hi - the links in the README to purchase the app in both the Google Play store and Amazon Appstore are broken, and I couldn't find it in either by searching. Is the app still available in either of these stores? If so what are the current links? Or if not what is the recommended way to get it? Thanks :slightly_smiling_face:

**Comment by robinrosenstock** [comment]
haven't found it either.
But it seems that [simpletask-android](https://github.com/mpcjanssen/simpletask-android) could become the new canonical one. It is [here on the playstore](https://play.google.com/store/apps/details?id=nl.mpcjanssen.simpletask&hl=en)

**Comment by bobwhitelock** [comment]
@geniusupgrader yeah that's what I found after creating this issue and settled on using; would be nice to try out both apps if both are available though :slightly_smiling_face:

**Comment by akalongman** [comment]
Same question

**Comment by adityabhaskar** [comment]
The original official app was removed from the Play store a long while ago. It had stopped working after Dropbox updated their API. There are quite a few third party apps available now, listed under '`Community Apps > Mobile`' on the [todotxt homepage](http://todotxt.org/).

I would suggest trying my app, available on Play Store for most Android devices:  [Todo.txt for Android](https://play.google.com/store/apps/details?id=net.c306.ttsuper&referrer=utm_source%3Dtodotxt_website%26utm_medium%3Dcommunity_apps&utm_source=todotxt_github&utm_medium=app_store)
